src/ResourceManagement/RecoveryServices/RecoveryServicesManagement/Microsoft.Azure.Management.RecoveryServices.nuget.proj

```diff
@@ -5,7 +5,7 @@
     Microsoft.Azure.Management.RecoveryServices
     -->
     <SdkNuGetPackage Include="Microsoft.Azure.Management.RecoveryServices">
-      <PackageVersion>1.0.2-preview</PackageVersion>
+      <PackageVersion>1.0.3-preview</PackageVersion>
```

Comment:
@amitmall987 You have changed client names in hydra specs. this is a breaking change. Please bump up the major version.